streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -603,7 +606,7 @@ boolean runLoop() {
                     log.error("Shutting down because the Kafka cluster seems to be on a too old version. " +
                               "Setting {}=\"{}\" requires broker version 2.5 or higher.",
                           StreamsConfig.PROCESSING_GUARANTEE_CONFIG,
-                          EXACTLY_ONCE_BETA);
+                          StreamsConfig.EXACTLY_ONCE_V2);
```

Comment:
We don't know if the user set `_beta` or `_v2` -- should we try to get the value from `StreamsConfig` to provide an error message that matches whatever the user specified?

Reply:
My rationale was that if a user is hitting this then it's presumably a new application that they just tried to enable eos-v2 with, without upgrading their brokers. And if it's a new 3.0 application then why would they have chosen the deprecated eos-beta config over the eos-v2 config.

But I can try to print the actual result, it just requires a little extra code to look this up and save it since we don't track the configs after the thread's creation. Not the end of the world, but didn't feel necessary to me. Thoughts?

Reply:
SGTM -- if the required code changes to get the actual value are to much, I am fine with hard-coding the value, too.

Reply:
I'll just put in both of them